ts/navigation/AppStackNavigator.tsx

```diff
@@ -199,6 +210,16 @@ const InnerNavigationContainer = (props: { children: React.ReactElement }) => {
               "card-onboarding-attempts",
             ...(bpdEnabled
               ? { [BPD_ROUTES.CTA_BPD_IBAN_EDIT]: "bpd-iban-update" }
+              : {}),
```

Comment:
These ternary operator seems to be used only in this Navigator. Should we the other syntax `...(bpdEnabled && { [BPD_ROUTES.CTA_BPD_IBAN_EDIT]: "bpd-iban-update" }`, that is used on every navigator?

Reply:
Same as comment on 213

Reply:
actually this object is used to define the internal link paths, so the fact is: if the condition is verified we need to build the custom link otherwise nothing should be add in the config. Were you thinking to any other kind of solution?

Reply:
We think that writing

```
...(bpdEnabled
              ? { [BPD_ROUTES.CTA_BPD_IBAN_EDIT]: "bpd-iban-update" }
              : {})
```
              
is the same as writing

`...(bpdEnabled && { [BPD_ROUTES.CTA_BPD_IBAN_EDIT]: "bpd-iban-update" })`

While the final result is the same, the latter syntax is used on every Navigator in the application code, so maybe is better to re-write the former just to have uniformed syntax.

Reply:
fixed in [c6112ed](https://github.com/pagopa/io-app/pull/4164/commits/c6112ed3cefee779a5d2d94bb70a0b1acdaaa81f)